### ISSUE TITLE
How to have a night LED mode (timer) for router's led to turn off and on all leds at particular times daily on openwrt firmware? 

### ISSUE THREAD

**Issue by blue-pine** [openwrt/openwrt#14279]
### Describe the bug

I tried myself and also searched everywhere for setting night led mode activated for OpenWrt firmware but all effort went to no avail for such a simple feature. This is already avilable on the stock firmware of my wifi router TP-Link Archer C6 AC1200 v3.2 which is itself based on OpenWrt and for which I have provide all related files from my research here in my repository : https://github.com/blue-pine/openwrt-tplink_archer_c6_v3.2 . Still I have failed to join the led_night_mode library (from stock router firmware) to the OpenWrt firmware for use on the router. So, I ask for everyone's help whatever they can provide for getting this last and only thing lacking on all OpenWrt firmwares. If you need any help then please ask me, you are always welcome.
Thank you

### OpenWrt version

all versions

### OpenWrt release

all releases

### OpenWrt target/subtarget

all targets

### Device

TP-Link Archer C6 AC1200 v3.2

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

**Comment by brada4** [comment]
start with 'uci export' and flipp trigger to off in all 'lonfig led' sections. Then trigger activity via main crontab.
not a bug like to change code. Likely someone smart on forums can teach you script tricks

**Comment by blue-pine** [comment]
> start with 'uci export' and flipp trigger to off in all 'lonfig led' sections. Then trigger activity via main crontab. not a bug like to change code. Likely someone smart on forums can teach you script tricks

Thank you, but I am a beginner and dont know how to go forward, so how can I integrate this timer with openwrt ?

**Comment by brada4** [comment]
Yes, just present your idea in forums, it is not defect per se, it is a quite reasonable wishlist item for new or changing existing wifischedule (similar in the way that it shuts down then restores radios on schedules) package

**Comment by blue-pine** [comment]
> Yes, just present your idea in forums, it is not defect per se, it is a quite reasonable wishlist item for new or changing existing wifischedule (similar in the way that it shuts down then restores radios on schedules) package

Yes, grateful for your help
Thank you

**Comment by raenye** [comment]
`wifischedule` package does something similar for WLANs.
Maybe it is time for a more comprehensive "event scheduler" OpenWrt package.

**Comment by blue-pine** [comment]
> `wifischedule` package does something similar for WLANs. Maybe it is time for a more comprehensive "event scheduler" OpenWrt package.

Yes, but wifischeduler sets timer only for the wifi radio not for led indicators. So features like led night timer highlight the limitation and complications of ths package with respect to use of openwrt on wifi routers

**Comment by raenye** [comment]
I'm not saying it does what you want. But (a) it has similar functionality if you want to learn about using the cron scheduler; and (b) someone could extend it to a general scheduler framework.

**Comment by blue-pine** [comment]
> I'm not saying it does what you want. But (a) it has similar functionality if you want to learn about using the cron scheduler; and (b) someone could extend it to a general scheduler framework.

Right. Thanks for your explanation. I appreciate it
